cmd/notable-cli/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
The license(s) of the current dependencies need to be evaluated (on my todo list). This project is currently MIT licensed but I don't know if that's appropriate (or Apache 2 for the same reason).